app/models/concerns/thredded/post_common.rb

```diff
@@ -59,37 +58,22 @@ def context_options
     end
 
     def sanitize_whitelist
-      HTML::Pipeline::SanitizationFilter::WHITELIST.deep_merge(
-        attributes: {
-          'code'       => ['class'],
-          'img'        => %w(src class width height),
-          'blockquote' => ['class'],
-        }
+      HTML::Pipeline::SanitizationFilter::WHITELIST[:elements] += %w(
+        fieldset
+        legend
+        blockquote
       )
-    end
-
-    def html_filter_for_pipeline
-      if filter == 'bbcode'
-        HTML::Pipeline::BbcodeFilter
-      else
-        HTML::Pipeline::MarkdownFilter
-      end
+      HTML::Pipeline::SanitizationFilter::WHITELIST
```

Comment:
> blockquote is already in the whitelist, and it doesn't seem that fieldset/legend should be whitelisted.

I think you're right. Markdown will just wrap a blockquote, right? If that's the case then, yeah - we should retain some parity there.  Will inspect this html once I submit :)